catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ValidateCommand.java

```diff
@@ -150,7 +161,11 @@ public Object execute() throws Exception {
         if (file.isFile()) {
             files = Collections.singletonList(file);
         } else {
-            files = FileUtils.listFiles(file, null, true);
+            String[] extensions = null;
+            if (filteredExtensions != null && !filteredExtensions.isEmpty()) {
```

Comment:
I also think I'm going to revisit this to make it less confusing. I don't see the list being empty since it's filled from the command line. I think it'll only be null or have contents.